src/main/java/BadPracticesModel.java

```diff
@@ -0,0 +1,68 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
Multiple Lombok annotations are used that may overlap in functionality. Specifically, @Data already includes @Getter, @Setter, @ToString, and @EqualsAndHashCode. (Severity: low)
**Suggestion**: Consider using only @Data or selectively using the necessary annotations to reduce redundancy.